manifests/ocp/crd.projectcalico.org_bgpconfigurations.yaml

```diff
@@ -1,3 +1,5 @@
+---
```

Comment:
I'm not sure why it's adding these extra `---` and comments, need to look into whether helm can be configured not to do that.